DataFormats/Headers/include/Headers/DataHeader.h

```diff
@@ -215,33 +200,42 @@ constexpr T String2(const char (&str)[N])
 /// have to be studied.
 template <int N, typename PrinterPolicy = Internal::defaultPrinter>
 struct Descriptor {
-  static_assert(Internal::NumberOfActiveBits<N>::value == 1 && N <= 8,
+  static_assert(Internal::NumberOfActiveBits<N>::value == 1,
 		"size is required to be 1 or a multiple of 2");
```

Comment:
thanks, yes, that would be the correct error message and it should be changed. It's supposed to be a power of two, i.e. only one bit set. The previous "multiple of 2" message was wrong.

PS: I like the `(N & (N - 1)) == 0`-logic, it's more fancy but in terms of readability of the code one would need to add a comment what it boils down to.